src/_empty.scss

```diff
@@ -1,18 +1,18 @@
 // Empty states (or Blank slates)
 .empty {
-  background: @core-empty-bg;
+  background: #f8f8f8;
   border-radius: .3rem;
   text-align: center;
-  padding: 4 * @layout-gutter;
+  padding: 2 * $layout-gutter;
 
   .empty-title {
     font-size: 1.8rem;
     margin: 1.5rem 0 .5rem 0;
   }
   .empty-meta {
-    color: @core-meta-color;
+    color: $core-meta-color;
   }
-  .empty-action {
+  .btn {
```

Comment:
This should be left as .empty-action as it breaks the inline form with a button. 

![screen shot 2016-06-04 at 17 02 30](https://cloud.githubusercontent.com/assets/6320793/15800627/2d9ecd2e-2a76-11e6-84a4-eb80cbb8ef0c.jpg)